### PR TITLE
:mortar_board: Assignment tsp --- Describe latex 

### PR DESCRIPTION
**Pull request by jameshughes89**
### What

Talk about how they can use latex for more points

### How

Kept high-level as I want them to figure it out.

### Testing

:+1: 

### Additional Notes

I used to include ACM's template too, but people tend to find the IEEE one much easier to work with. Also, the reason for the conference template instead of the journal one is that it includes less weird things (like it asks for author bios and pics which will just throw off the students).

### CODE REVIEW

File: site/assignments/tsp/tsp.rst [hunk 1]
```diff
@@ -278,10 +278,27 @@ and is not intended to be the standard template.
...
 LaTeX
 -----
 
+An additional **2** points may be obtained if the report is written in LaTeX.
+
+LaTeX is powerful software for writing and typesetting documents. Everything is written in plain text with various tags
+that LaTeX will use to format the document nicely.
+
+Although it is possible to download, write, and build everything locally on a personal computer, it is highly
+recommended to use `Overleaf <https://www.overleaf.com/>`_. Overleaf is an online editor that takes care a lot of
```
Comment (twentylemon) [inline site/assignments/tsp/tsp.rst:290]: yo I also main overleaf

File: site/assignments/tsp/tsp.rst [hunk 2]
```diff
@@ -278,10 +278,27 @@ and is not intended to be the standard template.
...
+tedious setup and it automatically backs up all work.
+
+If using LaTeX, the report must be written with the
+`IEEE <https://www.overleaf.com/latex/templates/ieee-conference-template/grfzhhncsfqn>`_  conference template. Overleaf
+makes it simple to start using the template.
```
Comment (twentylemon) [inline site/assignments/tsp/tsp.rst:295]: Consider maybe dropping this as a hard requirement, at least for assignment one. Templates might be overwhelming as they throw loads of code at you.

File: site/assignments/tsp/tsp.rst (hunk 1 above)
Comment (jameshughes89) [reply site/assignments/tsp/tsp.rst:290]: I resisted until like 2017 and then wondered wtf was wrong with me.

File: site/assignments/tsp/tsp.rst (hunk 2 above)
Comment (jameshughes89) [reply site/assignments/tsp/tsp.rst:295]: good call.

**Pull request merged**